lib/ansible/modules/packaging/os/zypper_repository.py

```diff
@@ -107,6 +114,13 @@
     repo: 'ftp://download.nvidia.com/opensuse/12.2'
     state: absent
 
+# Add NVIDIA repository for graphics drivers with additional arguments
+- zypper_repository:
+    name: nvidia-repo
+    repo: 'ftp://download.nvidia.com/opensuse/12.2'
```

Comment:
No need for quotes here, we only tend to add quotes where they are required by YAML.
This is the best way to ensure Ansible users learn when quotes are really required and when not.
```suggestion
    repo: ftp://download.nvidia.com/opensuse/12.2
```
Please modify the other examples as well :-)

Reply:
Done